GitUI/CommandsDialogs/SettingsDialog/Pages/ConfirmationsSettingsPage.Designer.cs

```diff
@@ -28,346 +28,383 @@ protected override void Dispose(bool disposing)
+            chkSecondAbortConfirmation.Text = "Confirm for the second time to abort a merge";
+            chkSecondAbortConfirmation.ThreeState = true;
+            chkSecondAbortConfirmation.UseVisualStyleBackColor = true;
             // 
             // chkUpdateModules
             // 
-            this.chkUpdateModules.AutoSize = true;
-            this.chkUpdateModules.Location = new System.Drawing.Point(3, 476);
-            this.chkUpdateModules.Name = "chkUpdateModules";
-            this.chkUpdateModules.Size = new System.Drawing.Size(201, 19);
-            this.chkUpdateModules.Text = "Update submodules on checkout";
-            this.chkUpdateModules.ThreeState = true;
-            this.chkUpdateModules.UseVisualStyleBackColor = true;
+            chkUpdateModules.AutoSize = true;
+            chkUpdateModules.Location = new Point(3, 501);
+            chkUpdateModules.Name = "chkUpdateModules";
+            chkUpdateModules.Size = new Size(201, 19);
+            chkUpdateModules.TabIndex = 19;
+            chkUpdateModules.Text = "Update submodules on checkout";
+            chkUpdateModules.ThreeState = true;
+            chkUpdateModules.UseVisualStyleBackColor = true;
             // 
             // chkSwitchWorktree
             // 
-            this.chkSwitchWorktree.AutoSize = true;
-            this.chkSwitchWorktree.Location = new System.Drawing.Point(3, 528);
-            this.chkSwitchWorktree.Name = "chkSwitchWorktree";
-            this.chkSwitchWorktree.Size = new System.Drawing.Size(110, 19);
-            this.chkSwitchWorktree.Text = "Switch worktree";
-            this.chkSwitchWorktree.UseVisualStyleBackColor = true;
+            chkSwitchWorktree.AutoSize = true;
+            chkSwitchWorktree.Location = new Point(3, 553);
+            chkSwitchWorktree.Name = "chkSwitchWorktree";
+            chkSwitchWorktree.Size = new Size(110, 19);
+            chkSwitchWorktree.TabIndex = 21;
+            chkSwitchWorktree.Text = "Switch worktree";
+            chkSwitchWorktree.UseVisualStyleBackColor = true;
             // 
-            // chkBranchDeleteUnmerged
+            // chkBranchCheckoutConformation
             // 
-            this.chkBranchDeleteUnmerged.AutoSize = true;
-            this.chkBranchDeleteUnmerged.Location = new System.Drawing.Point(3, 220);
-            this.chkBranchDeleteUnmerged.Name = "chkBranchDeleteUnmerged";
-            this.chkBranchDeleteUnmerged.Size = new System.Drawing.Size(168, 19);
-            this.chkBranchDeleteUnmerged.Text = "Delete unmerged branches";
-            this.chkBranchDeleteUnmerged.UseVisualStyleBackColor = true;
+            chkBranchCheckoutConformation.AutoSize = true;
+            chkBranchCheckoutConformation.Location = new Point(3, 245);
+            chkBranchCheckoutConformation.Name = "chkBranchCheckoutConformation";
+            chkBranchCheckoutConformation.Size = new Size(282, 19);
+            chkBranchCheckoutConformation.TabIndex = 22;
```

Comment:
TabIndex was added, this is now incorrect.
Also move to the position in the form